<launch>
    <node pkg="djh_ins" name="djh_ins_node" type="djh_ins_node" output="screen">
        <!-- Default parameter settings -->
        <rosparam file="$(find djh_ins)/param/IMUmodel.yaml"/>
        <rosparam file="$(find djh_ins)/param/Earthparam.yaml"/>
    </node>

    <node pkg="djh_ins" name="bare_bones_node" type="bare_bones_node" output="screen">
        <!-- Default parameter settings -->
        <!-- NONE RIGHT NOW -->
    </node>

</launch>